nixos/modules/services/monitoring/grafana.nix

```
@@ -364,9 +364,15 @@ in {
             };
 
             http_addr = mkOption {
-              description = lib.mdDoc "Listening address.";
-              default = "";
               type = types.str;
+              default = "127.0.0.1";
+              description = lib.mdDoc ''
+                Listening address.
+
+                ::: {.note}
+                This setting intentionally varies from upstreams default to be a bit securer by default.
```

more secure

---

```suggestion
                This setting intentionally varies from upstream's default to be a bit more secure by default.
```

---

I was thinking:

```suggestion
                This setting intentionally varies from upstreams default to be more secure by default.
```

but idk

---

I was thinking:

```suggestion
                This setting intentionally varies from upstreams default to be more secure by default.
```

but idk